src/components/ec-table-sort/ec-table-sort.vue

```diff
@@ -70,33 +70,31 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/accessibility';
+<style>
 
 .ec-table-sort {
-  line-height: 0;
-  user-select: none;
-  cursor: pointer;
+  @apply tw-select-none;
+  @apply tw-cursor-pointer;
+
   color: inherit;
 
   &:hover,
   &:focus,
   &:active {
-    color: $level-3-hover;
-    outline: 0;
+    @apply tw-text-key-3;
+    @apply tw-outline-none;
   }
 
   &__icon {
-    fill: currentColor;
+    @apply tw-fill-current;
 
     &--asc {
       transform: rotateX(180deg);
```

Comment:
I guess we can add some class for theses cases

Reply:
Yep, I checked and we only have this instance so far. Maybe in the future?